apps/astarte_data_updater_plant/mix.exs

```diff
@@ -84,7 +84,8 @@ defmodule Astarte.DataUpdaterPlant.Mixfile do
       {:skogsra, "~> 2.2"},
       {:telemetry, "~> 0.4"},
       {:observer_cli, "~> 1.5"},
-      {:dialyzex, github: "Comcast/dialyzex", only: [:dev, :ci]}
+      {:dialyzex, github: "Comcast/dialyzex", only: [:dev, :ci]},
+      {:ex_rabbit_pool, github: "leductam/ex_rabbit_pool"}
```

Comment:
Actually it's because the esl repo does not even support `amqp` version `2.1`, while leductam's fork only change is supporting also `amqp ~> 2.0` and `~> 3.0`